docs/getting-started-guides/juju.md

```diff
@@ -1,10 +1,12 @@
 ---
 ---
 
-[Juju](https://jujucharms.com/docs/stable/about-juju) makes it easy to deploy
-Kubernetes by provisioning, installing and configuring all the systems in
-the cluster. Once deployed the cluster can easily scale up with one command
-to increase the cluster size.
+[Juju](https://jujucharms.com/docs/2.0/about-juju) encapsulates the 
+operational knowledge of provisioning, installing, and securing a Kubernetes
+cluster into one step. Juju allows you to consistently deploy a cluster on
+different cloud providers with a consistent repeatable open source operations.
```

Comment:
Nit: You've mentioned consistent three times in the same paragraph. Please revise